CHANGELOG

```diff
@@ -1,3 +1,8 @@
+Version 1.4.0 (2021-11-17)
```

Comment:
Let's keep the CHANGELOG out of this PR.
I'll update it with the release of the next version.

Reply:
ok!